README.md

```diff
@@ -1,6 +1,7 @@
 # Microsoft Authentication CLI
 
-[![Tests](https://shields.io/github/workflow/status/AzureAD/microsoft-authentication-cli/Build%20and%20Test/main?style=for-the-badge&logo=github)](https://github.com/AzureAD/microsoft-authentication-cli/actions/workflows/dotnet-test.yml)
+
```

Comment:
What's up with the extra newline?

Reply:
I'm not sure - there is still only 1 newline in VS Code between lines 3 and 4

Reply:
Oh I added 1 above